requirements.txt

```diff
@@ -0,0 +1,2 @@
+flywheel-sdk>=12.0.0
+pydicom>=1.1.0
```

Comment:
Let's use: `pydicom~=1.4.2`
This version solved some issues we've seen with various DICOMs.